local_gemma/modeling_local_gemma_2.py

```diff
@@ -168,5 +173,9 @@ def from_pretrained(
         if device not in str(model.device) and preset_kwargs.get("device_map", None) is None:
             # for consistent behaviour with bitsandbytes, we move the model to the device always
             model.to(device, dtype=preset_kwargs["torch_dtype"])
+    
+        if quantization_config is not None and device == "cuda" and quantization == "hqq":
+            from hqq.utils.patching import prepare_for_inference
```

Comment:
We should gate the `hqq` import - it should only be imported if `is_hqq_available`